artbutton.js

```diff
@@ -380,11 +415,18 @@ class MultiMediaPopout extends ImagePopout {
 	}
 }
 
-Hooks.once("init", ShowArt.registerBindings.bind(ShowArt));
+Hooks.once("init", () => {
+	const registerKeybinds = ShowArt.registerBindings.bind(ShowArt);
+	registerKeybinds();
+
+	const registerSettings = ShowArt.registerConfig.bind(ShowArt);
+	registerSettings();
+});
```

Comment:
I would just add this as another line like:
```js
Hooks.once("init", ShowArt.registerConfig.bind(ShowArt));
```

The reason for the use of `bind` here is that `Hooks.once` take a function as the second argument, but the static methods of `ShowArt` can't be passed to it on their own, they need a `this` reference. But if you intend to directly call those functions, instead of passing them into the hook in order for it call later, you can just do:

```js
Hooks.once("init", () => {
    ShowArt.registerBindings();
    ShowArt.registerConfig();
});
```
Here, the function being passed in is the arrow function `() =>` so that function doesn't need a `this` bound to it. And since we can call the static methods of `ShowArt` directly, they are already bound to `ShowArt` here. 

Just JavaScript nonsense 🙄